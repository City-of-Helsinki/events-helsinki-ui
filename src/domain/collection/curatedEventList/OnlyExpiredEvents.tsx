import React from "react";
import { useTranslation } from "react-i18next";

import Link from "../../../common/components/link/Link";
import useLocale from "../../../hooks/useLocale";
import { ROUTES } from "../../app/constants";

const OnlyExpiredEvents = () => {
  const locale = useLocale();
  const { t } = useTranslation();

  return (
    <div>
      <p>{t("collection.curatedEvents.textOnlyExpiredEvents")}</p>
      <Link to={`/${locale}${ROUTES.COLLECTIONS}`}>
        {t("collection.curatedEvents.linkOnlyExpiredEvents")}
      </Link>
    </div>
  );
};
export default OnlyExpiredEvents;
