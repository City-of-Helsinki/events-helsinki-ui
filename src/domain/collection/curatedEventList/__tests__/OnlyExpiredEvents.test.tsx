import { render } from '@testing-library/react';
import React from 'react';
import { MemoryRouter } from 'react-router';

import OnlyExpiredEvents from '../OnlyExpiredEvents';

it('matches snapshot', () => {
  const { container } = render(
    <MemoryRouter>
      <OnlyExpiredEvents />
    </MemoryRouter>
  );

  expect(container.firstChild).toMatchSnapshot();
});
