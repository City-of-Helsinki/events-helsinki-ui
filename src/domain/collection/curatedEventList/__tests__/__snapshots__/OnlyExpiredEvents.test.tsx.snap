// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <p>
    Valitettavasti tämän kokoelman kaikki suositellut tapahtumat ovat jo ehtineet päättyä. Käy tutustumassa muihin suosituksiimme ja löydä itsellesi tekemistä.
  </p>
  <a
    class="link defaultColor defaultSize"
    href="/fi/collections"
  >
    Tutustu kaikkiin kokoelmiin
    <svg
      class="Icon-module_icon__1Jtzj icon_hds-icon__1YqNC Icon-module_s__2WGWe icon_hds-icon--size-s__2Lkik"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fill-rule="evenodd"
      >
        <path
          d="M0 24V0h24v24z"
        />
        <path
          d="M13 12.5l-5-5L9.5 6l6.5 6.5L9.5 19 8 17.5z"
          fill="currentColor"
        />
      </g>
    </svg>
  </a>
</div>
`;
